Dockerfile

```diff
@@ -2,13 +2,16 @@ FROM node:12.9.0
 
 WORKDIR /
 
+ARG REPORT_INPUT
+
 ENV ACTION_NAME='./actions-npm-audit/'
 
+ADD $REPORT_INPUT .
```

Comment:
Was there any reason that `ADD` was selected over `COPY`? `COPY`, in general, should be used as the best practice since it's more explicit.

Reply:
ADD was replaced by COPY